src/effects/music_note.c

```diff
@@ -2,13 +2,110 @@
 #include "effects_internal.h"
 
 void music_note_appendGfx(void* effect);
+void music_note_init(EffectInstance* effect);
+void music_note_update(EffectInstance* effect);
+void music_note_render(EffectInstance* effect);
 
-INCLUDE_ASM(s32, "effects/music_note", music_note_main);
+s32 D_E004C660[] = { 0x09001038, 0x090010C0, 0x09001148, 0x090011D0, 0x09001258, 0x090012E0, 0x09001368 };
```

Comment:
These are display list pointers. can you do the thing we do in big_smoke_puff, for example?

```
extern Gfx D_09000FA0[];
extern Gfx D_09001060[];
extern Gfx D_09001120[];
extern Gfx D_090011E0[];
extern Gfx D_090012A0[];
extern Gfx D_09001360[];
extern Gfx D_09001420[];

Gfx* asdf[7] = {
    D_09000FA0, D_09001060, D_09001120, D_090011E0, D_090012A0, D_09001360, D_09001420
};
```
and then add the 0900 syms to undefined_syms.txt